iina/config/vlc-default-input.conf

```diff
@@ -26,8 +26,8 @@ Shift+Meta+RIGHT seek 60
 Shift+Alt+Meta+LEFT seek -300
 Shift+Alt+Meta+RIGHT seek 300
 e frame-step
-Meta+L ab-loop
-Meta+l cycle-values loop "inf" "no"
+Meta+l ab-loop
```

Comment:
I'm thinking this is the wrong fix. IINA should match the VLC key bindings. When we choose to extend the VLC key bindings to provide access to IINA features like the log viewer, that key binding must avoid conflicting with the bindings defined by VLC. It is more important to match VLC than use the same key binding for IINA features in the different defaults.